en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md

```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
-            <property expression="json-eval($.clientNamespace)" name="clientNamespace" scope="default" type="STRING"/>
-            <property expression="json-eval($.apiVersion)" name="apiVersion" scope="default" type="STRING"/>
-            <property expression="json-eval($.contextDateRangeFrom)" name="contextDateRangeFrom" scope="default" type="STRING"/>
-            <property expression="json-eval($.contextDateRangeTo)" name="contextDateRangeTo" scope="default" type="STRING"/>
-            <property expression="json-eval($.xRefCode)" name="xRefCode" scope="default" type="STRING"/>
-            <ceridiandayforce.init>
-                <username>{$ctx:username}</username>
-                <password>{$ctx:password}</password>
-                <clientNamespace>{$ctx:clientNamespace}</clientNamespace>
-                <apiVersion>{$ctx:apiVersion}</apiVersion>
-            </ceridiandayforce.init>
-            <ceridiandayforce.getEmployeeAddresses>
-                <xRefCode>{$ctx:xRefCode}</xRefCode>
-                <contextDate>{$ctx:contextDate}</contextDate>
-                <contextDateRangeFrom>{$ctx:contextDateRangeFrom}</contextDateRangeFrom>
-                <contextDateRangeTo>{$ctx:contextDateRangeTo}</contextDateRangeTo>
-            </ceridiandayforce.getEmployeeAddresses>
-            <send/>
-        </inSequence>
-        <outSequence/>
-        <faultSequence/>
-    </resource>
-    <resource methods="POST" uri-template="/postEmployeeContact">
-        <inSequence>
-         <log level="full" separator=","/>
-         <property expression="json-eval($.username)" name="username"/>
-         <property expression="json-eval($.password)" name="password"/>
-         <property expression="json-eval($.clientNamespace)" name="clientNamespace"/>
-         <property expression="json-eval($.apiVersion)" name="apiVersion"/>
-         <property expression="json-eval($.isValidateOnly)" name="isValidateOnly"/>
-         <property expression="json-eval($.fieldAndValue)" name="fieldAndValue"/>
-         <property expression="json-eval($.xRefCode)" name="xRefCode"/>
-         <ceridiandayforce.init>
-            <username>{$ctx:username}</username>
-            <password>{$ctx:password}</password>
-            <clientNamespace>{$ctx:clientNamespace}</clientNamespace>
-            <apiVersion>{$ctx:apiVersion}</apiVersion>
-         </ceridiandayforce.init>
-         <ceridiandayforce.postEmployeeContacts>
-            <xRefCode>{$ctx:xRefCode}</xRefCode>
-            <isValidateOnly>{$ctx:isValidateOnly}</isValidateOnly>
-            <fieldAndValue>{$ctx:fieldAndValue}</fieldAndValue>
-         </ceridiandayforce.postEmployeeContacts>
-         <send/>
-      </inSequence>
-        <outSequence/>
-        <faultSequence/>
-    </resource>
-</api>
-```
+Dayforce sample APIs. Keep them saved for use in the next steps.  
+
+## Setting up the integration project
```

Comment:
```suggestion
## Set up the integration project
```

Reply:
In the guideline, it has been suggested to use the `-ing` form.

Reply:
The guideline has been revised to use the simple present tense.